x/exchange/client/cli/tx.go

```diff
@@ -169,17 +169,17 @@ func CmdTxMarketUpdateDetails() *cobra.Command {
 	return cmd
 }
 
-// CmdTxMarketUpdateEnabled creates the market-enabled sub-command for the exchange tx command.
+// CmdTxMarketUpdateEnabled creates the market-accepting-orders sub-command for the exchange tx command.
 func CmdTxMarketUpdateEnabled() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "market-enabled",
-		Aliases: []string{"market-update-enabled", "update-market-enabled", "update-enabled"},
+		Use:     "market-accepting-orders",
+		Aliases: []string{"market-update-accepting-orders", "update-market-accepting-orders", "update-accepting-orders"},
```

Comment:
Do we want to add market-enabled as an alias in case anyone is using it? I doubt they are, but might be helpful just in case.

Reply:
I thought about that, and decided that we'd be okay without it. To begin with, I wanted as little "enabled/disabled" wording left around as possible in order to help prevent future confusion. But because the exchange stuff is so new, and no one's really using it yet, combined with the rarity of using this specific command, I felt it was okay to not keep the old stuff here.